src/main/java/org/folio/oaipmh/helpers/storage/StorageHelper.java

```diff
@@ -21,8 +17,6 @@ static StorageHelper getInstance() {
     String repositoryType = System.getProperty(REPOSITORY_STORAGE, SOURCE_RECORD_STORAGE);
     if (SOURCE_RECORD_STORAGE.equals(repositoryType)) {
       return new SourceRecordStorageHelper();
-    } else if (INVENTORY_STORAGE.equals(repositoryType)) {
```

Comment:
I'm thinking that maybe it would be better to default to SRS and log a warning instead of removing this completely.  Doing it this way means this is a breaking change and anyone using this option would need to explicitly change it to get the module running again.

Either way, we should probably note this in the goldenrod release notes.  I don't *think* anyone is using this option, but who knows.

Reply:
And really, if we're only going to support this one storage, we can probably clean a lot of this up - Maybe track this in a story tagged as tech-debt?

Reply:
This inventory-storage has been removed because it is a) not used from the business requirements perspective b) we have implemented another means of communicating with inventory c) this class was hard to maintain, was obsolete and entailed writing/supporting tests  for it.
So for metadata prefix marc21_withholdings we use another method of getting instances from inventory, inventory-view. And for all other metadata prefixes we do not use inventory at all.

Reply:
oh, interesting - so the ability to do this is temporarily removed, but slated to be re-added in Q3?

Reply:
It will be a completely different mechanism that talks to inventory in Q3, see https://github.com/folio-org/mod-oai-pmh/blob/master/src/main/java/org/folio/oaipmh/processors/MarcWithHoldingsRequestHelper.java#L292